cmd/capdctl/main.go

```diff
@@ -68,12 +74,9 @@ func (mo *machineDeyploymentOptions) initFlags(fs *flag.FlagSet) {
 func main() {
 	setup := flag.NewFlagSet("setup", flag.ExitOnError)
 	managementClusterName := setup.String("cluster-name", "management", "The name of the management cluster")
-
-	// crds takes no args
-
-	capd := flag.NewFlagSet("capd", flag.ExitOnError)
-	capdImage := capd.String("capd-image", "gcr.io/kubernetes1-226021/capd-manager:latest", "The capd manager image to run")
-	capiImage := capd.String("capi-image", "gcr.io/k8s-cluster-api/cluster-api-controller:0.1.3", "The capi manager image to run")
+	version := setup.String("capi-version", "v0.1.4", "The CRD versions to pull from CAPI")
+	capdImage := setup.String("capd-image", "gcr.io/kubernetes1-226021/capd-manager:latest", "The capd manager image to run")
```

Comment:
Hi @chuckha,

I'm not personally a fan of `latest` as it has unpredictable results. Will using it here result in possible breakage?

Reply:
Totally agree. I've opened an issue to figure out a better strategy here: https://github.com/kubernetes-sigs/cluster-api-provider-docker/issues/67